pkg/ccl/testutilsccl/alter_primary_key.go

```diff
@@ -62,6 +62,9 @@ func AlterPrimaryKeyCorrectZoneConfigTest(
 		t.Run(tc.Desc, func(t *testing.T) {
 			var db *gosql.DB
 			var params base.TestServerArgs
+			params.DefaultTestTenant = base.TestDoesNotWorkWithSharedProcessModeButWeDontKnowWhyYet(
```

Comment:
once https://github.com/cockroachdb/cockroach/pull/114037 merges, this should not be skipped

Reply:
good point, I'll update after it's merged.